documentation/cli/docs/02_wallet.md

```diff
@@ -79,13 +79,13 @@ The wallet can only be initialised once.
 
 When just initialised, the wallet has no account yet, use the `new` command to create one.
 
-#### Parameters
+#### Options
 
-| Name        | Optional    | Default                             | Example                                                                                                                                                                             |
-| ----------- | ----------- |------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `mnemonic`  | ✓           | Randomly generated                  | "aunt middle impose faith ramp kid olive good practice motor grab ready group episode oven matrix silver rhythm avocado assume humble tiger shiver hurt" (DO NOT USE THIS MNEMONIC) |
-| `node`      | ✓           | https://api.testnet.shimmer.network | http://localhost:14265                                                                                                                                                              |
-| `coin-type` | ✓           | 4219 (=Shimmer)                     | 4218 (=IOTA)                                                                                                                                                                        |
+| Name                 | Description    | Default    | Environment Variable | Example |
+| -------------------- | -------------- |----------- |----------------------| ------- |
+| `--mnemonic` (`-m`)  | Initialises a wallet from an existing mnemonic | ✘ | ✘ |"aunt middle impose ..." |
+| `--node_url` (`-n`)  | Sets the node to connect to | https://api.testnet.shimmer.network | `NODE_URL` | http://localhost:14265 |
+| `--coin-type` (`-c`) | Sets the coin type associated with the wallet  | 4219 (=Shimmer) | ✘ | 4218(=IOTA) |
```

Comment:
Should this be node-url or coin_type?

Reply:
oh, nice catch...should be dashes in all cases.